CONTRIBUTING.md

```diff
@@ -1,9 +1,67 @@
-# How to contribute
+# Contributing guide
 
-Government employees, public and members of the private sector are encouraged to contribute to the repository by **forking and submitting a pull request**.
+Thanks for taking a moment and reading this guide. Is very important to have 
```

Comment:
I think an "it" is missing here, "Is very important..." -> "It is very important..."

Reply:
Yeah! Thanks for noticing. I fixed this item in the another PR #4 =D

Reply:
Sorry, I didn't noticed hehe.